MATLAB/rl_ofdm_dl_mimo.m

```diff
@@ -399,7 +396,7 @@
     end
 
     % Check if valid...
-    pk_tmp = preamble_pk(iue); 
+    pk_tmp = preamble_pk(iue) + golden_number;
```

Comment:
platinum?

Reply:
it was "magic" at first but it felt too cynical